specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceGetIndex.json

```diff
@@ -229,6 +229,15 @@
                         "tempuri.org"
                     ],
                     "maxAgeInSeconds": 60
+                },
+                "encryptionKey": {
+                    "keyVaultKeyName": "myKeyName",
+                    "keyVaultKeyVersion": "myKeyVersion",
+                    "keyVaultUri": "https://myKeyVault.vault.azure.net",
+                    "accessCredentials": {
```

Comment:
Wouldn't the entire `accessCredentials` come back as `null`?

Reply:
accessCredential as a whole is an optional property, but if one is provided, we will only censor the application secret. In cases where the access credentials is not provided, we assume fallback to using MSI instead. In the other example, I've also used a few instances of accessCredentials being null to give some examples of MSI being used.

Reply:
Got it, thanks.